pages/cloudflare/get-started.mdx

```diff
@@ -81,32 +81,12 @@ This step is optional since `@opennextjs/cloudflare` creates this file for you d
 Add a [`open-next.config.ts`](https://opennext.js.org/aws/config) file to the root directory of your Next.js app:
 
 ```ts
-import type { OpenNextConfig } from "@opennextjs/aws/types/open-next.js";
+import { defineCloudflareConfig } from "@opennextjs/cloudflare";
 import cache from "@opennextjs/cloudflare/kv-cache";
```

Comment:
Would be nice that we can import all from `"@opennextjs/cloudflare"` - would you mind creating a PR for that and update the docs accordingly?

Reply:
did you see @james-elicx's comment? https://github.com/opennextjs/opennextjs-cloudflare/pull/412#discussion_r1971412907

Reply:
yes

Reply:
done